packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json

```diff
@@ -311,9 +311,9 @@
   "hosting_tab_DOMAINS_table_cdn_state_ACTIVE": "Activé",
   "hosting_tab_DOMAINS_table_cdn_state_NONE": "Désactivé",
   "hosting_tab_DOMAINS_table_ssl_state_true": "Activé",
-  "hosting_tab_DOMAINS_tooltip_success": "Votre domaine {{domainName}} pointe correctement sur votre hébergement {{hosting}} avec l'IP {{ip}}",
-  "hosting_tab_DOMAINS_tooltip_info": "Votre domaine {{domainName}} ne pointe pas sur votre hébergement {{hosting}} avec l'IP {{ip}}",
-  "hosting_tab_DOMAINS_tooltip_error": "Votre domaine {{domainName}} ne pointe actuellement sur aucune adresse IP",
+  "hosting_tab_DOMAINS_diagnostique_tooltip_true": "Votre domaine {{ domainName }} pointe correctement sur votre hébergement {{ hosting }} avec l'IP {{ ip }}",
+  "hosting_tab_DOMAINS_diagnostique_tooltip_false": "Votre domaine {{ domainName }} ne pointe pas sur votre hébergement {{ hosting }} avec l'IP {{ ip }}",
+  "hosting_tab_DOMAINS_diagnostique_tooltip_undefined": "Votre domaine {{ domainName }} ne pointe actuellement sur aucune adresse IP",
```

Comment:
**nitpick:** `undefined` does not make sense. Can we use some meaningful wording? Maybe `default`?
```suggestion
  "hosting_tab_DOMAINS_diagnostique_tooltip_default": "Votre domaine {{ domainName }} ne pointe actuellement sur aucune adresse IP",
```

Reply:
In fact i use directly the status in API (after a toString()) so if the property is undefined i use _undefined key.
It's for avoid a fonction or constante for convert the status while there are only 3 possible.

Reply:
it's fixed